src/python/pants/backend/shell/target_types.py

```diff
@@ -389,6 +389,18 @@ class RunInSandboxArgumentsField(StringSequenceField):
     help = f"Extra arguments to pass into the `{RunInSandboxRunnableField.alias}` field."
 
 
+class RunInSandboxStdoutFilenameField(StringField):
+    alias = "stdout"
```

Comment:
I'm only 80% joking when I say that the fields should be named `1>` and `2>` respectively :smile:

Reply:
If only I weren't trying to steer people away from shell concepts :D